Dockerfile

```diff
@@ -12,20 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.13.15 as builder
+FROM --platform=$BUILDPLATFORM golang:1.13.15 as builder
+
+ARG STAGINGVERSION
+ARG BUILDPLATFORM
+ARG TARGETPLATFORM
+
 WORKDIR /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
 ADD . .
-RUN make gce-pd-driver
+# RUN mkdir -p bin \
```

Comment:
this is for comments?

Reply:
oh my bad! :(
will remove that in a follow up pr